content/blog/2021-01-28-survey-vaccines.Rmd

```diff
@@ -0,0 +1,354 @@
+---
+title: "Using the COVID-19 Symptom Survey to Track Vaccination Uptake and Sentiment in the United States"
+author: "Alex Reinhart, Esther Kim, Andy Garcia, Sarah LaRocca"
+date: 2021-01-27
+tags:
+  - COVIDcast
+  - R
+  - symptom surveys
+  - vaccine
+authors:
+  - alex
+  - ekim
+  - agarcia
+  - slarocca
+heroImage: /blog/images/blog-lg-vaccine-survey.jpg
+heroImageThumb: /blog/images/blog-thumb-vaccine-survey.jpg
+related:
+  - 2020-12-10-masks-public
+  - 2020-10-06-survey-wave-4
+  - 2020-08-26-fb-survey
+summary: |
+  Data from the Symptom Survey reveal how COVID-19 vaccinations are rolling out
+  across the nation.
+acknowledgements: |
+  Facebook's team, including Kelsey Mulcahy, Katherine Morris, Leanna Morinishi,
+  Kris Barkume, Sid Palani, and Patrick Xu, assisted with the analysis and
+  writing of this post. At Carnegie Mellon, Wichada La Motte-Kerr assisted in
+  the development and deployment of many of these survey questions.
+output:
+  blogdown::html_page:
+    toc: true
+---
+
+The COVID-19 Symptom Survey, which is conducted by the Delphi Group in
+partnership with Facebook Data for Good, is one of the largest COVID-19 data
+collection efforts in the United States. To date, over 16 million responses have
+been collected since its launch in April 2020. Using the Symptom Survey, we have
+been able to track and inform trends on [daily
+symptoms](https://delphi.cmu.edu/blog/2020/09/21/can-symptoms-surveys-improve-covid-19-forecasts/),
```

Comment:
not a fan of all the absolute links but for the new blogdown version, it would require the infrastructure changes from: https://github.com/cmu-delphi/www-main/pull/228

Reply:
since the build changes have been merged, I changed the links